deploy/lib/server_config.rb

```diff
@@ -556,6 +556,43 @@ def restart
     return true
   end
 
+  # implemented verified restart
+  def restart
+    verify = find_arg(['--verify'])
+    if verify==='false'
+      restart_basic
+    else
+      # defaults to verified restart
+      old_timestamp = go(%Q{http://#{@properties["ml.server"]}:8001/admin/v1/timestamp}, "get").body
+      restart_basic
+      retry_count = 0
+      retry_max = 5
+      retry_interval= 10
```

Comment:
I think at least one of these should be property-driven -- desired values may differ depending on the environment. Please add a property to default.properties and make use of it here.